num2words/base.py

```diff
@@ -109,7 +109,7 @@ def to_cardinal(self, value):
         words, num = self.clean(val)
         return self.title(out + words)
 
-    def float2tuple(self, value):
+    def float2tuple(self, value, precision_b=False):
```

Comment:
You don't need this change, self.precision is a property of your class, and you are using this change only in line 120 of Num2Word_TH :

`        pre, post, precision = self.float2tuple(number, precision_b=True)`

You could call self.precision in your Num2Word_TH

Reply:
Oh I didn't know that, thanks.